src/main/java/core/basesyntax/SimpleFileEditor.java

```diff
@@ -0,0 +1,242 @@
+            try {
+                System.out.println("Last modified : " + Files.getLastModifiedTime(path));
+                System.out.println(String.format("File size : %d bite.", Files.size(path)));
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    private void createFile(String[] wordsCommand) {
+        if (validate(wordsCommand)) {
+            String pathString = wordsCommand[1].substring(1, wordsCommand[1].length() - 1);
+            String fileNameString = wordsCommand[2].substring(1, wordsCommand[2].length() - 1);
+            Path path = Paths.get(pathString + "\\" + fileNameString);
+            if (!Files.exists(Paths.get(pathString))) {
+                System.out.println("Path not exists!");
+                return;
+            }
+            if (Files.exists(path)) {
+                System.out.println("File exists! Overwrite ?");
+                String answer = "";
+                while (!answer.equals(YES)) {
+                    Scanner scanner = new Scanner(System.in);
+                    answer = scanner.nextLine();
+                    if (answer.equals("yes")) {
+                        try {
+                            Files.delete(path);
+                            Files.createFile(path);
+                            return;
+                        } catch (IOException e) {
+                            e.printStackTrace();
+                        }
+                    } else if (answer.equals(NOT)) {
+                        return;
+                    } else {
+                        System.out.println("Incorrectly answer!");
+                    }
+                }
+            }
+            try {
+                Files.createFile(path);
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    private void readFile(String[] wordsCommand) {
+        if (validate(wordsCommand)) {
+            String pathString = wordsCommand[1].substring(1, wordsCommand[1].length() - 1);
+            String fileNameString = wordsCommand[2].substring(1, wordsCommand[2].length() - 1);
+            Path path = Paths.get(pathString + "\\" + fileNameString);
+            if (!Files.exists(Paths.get(pathString))) {
+                System.out.println("Path not exists!");
+                return;
+            }
+            if (!Files.exists(path)) {
+                System.out.println("File not exists!");
+            } else {
+                try (BufferedReader bufferedReader = Files.newBufferedReader(path)) {
+                    String line;
+                    while ((line = bufferedReader.readLine()) != null) {
+                        System.out.println(line);
+                    }
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
+                }
+            }
+        }
+    }
+
+    private boolean validate(String[] wordsCommand) {
```

Comment:
There are a lot of repeating logic in this method, try to make separate method out of this and remove redundant checks.